db/migrate/20190215171522_rename_macedonia.rb

```diff
@@ -0,0 +1,3 @@
+macedonia = WorldLocation.find_by(slug: "macedonia")
+macedonia.update(slug: "north-macedonia")
```

Comment:
Might be worth using `update!` here so we get an exception if it fails for whatever reason.

Reply:
Right